packages/sync/src/Modules.php

```diff
@@ -94,19 +123,41 @@ static function initialize_modules() {
 		return array_map( array( 'Automattic\\Jetpack\\Sync\\Modules', 'set_module_defaults' ), $modules );
 	}
 
-	static function load_module( $module_class ) {
+	/**
+	 * Returns an instance of the given module class.
+	 *
+	 * @param string $module_class The classname of a Jetpack sync module.
+	 *
+	 * @return \Automattic\Jetpack\Sync\Modules\Module
```

Comment:
Can we remove the leading `\`? Those theoretically could cause difficulties with testing so I don't want to introduce them anywhere.
```suggestion
	 * @return Automattic\Jetpack\Sync\Modules\Module
```